integrations/tensorflow/iree_tf_compiler/iree-import-tflite-main.cpp

```diff
@@ -66,6 +67,8 @@ int main(int argc, char **argv) {
   registry.insert<quant::QuantizationDialect>();
   registry.insert<StandardOpsDialect, mlir::arith::ArithmeticDialect>();
 
+  RegisterAllTensorFlowDialects(registry);
```

Comment:
:cry: I think this is going to add a bunch of dependencies to building iree-import-tflite, which happens to be something we now have to build as part of the benchmark pipeline

Reply:
Do we build the iree-import-tf as well? I didn't worry about adding the dependency because I assumed we built both anyway. If not we could split tflite and tflite-with-flex ops as two separate import tools.

Reply:
We don't build iree-import-tflite as part of benchmarking. Just iree-import-tf. The integrations build builds both. It might be nice to split them out, but then again that's one more level of complication. I feel like the real answer would be layering this in such a way that we could extract the flex ops in a layer on top and then feed it to a normal tool. I'm fine with whichever option you decide